src/Services/SmartProxy.js

```diff
@@ -37,6 +37,9 @@ export const SmartProxyApi = createApi({
       query: ({ id }) => `v2/cluster/${id}/info`,
       transformResponse: (response) => response?.cluster,
     }),
+    getUpgradeRisks: builder.query({
+      query: ({ id }) => `v1/cluster/${id}/upgrade-risks-prediction`,
```

Comment:
I think upgrade risk predictions is under v2

Reply:
it is